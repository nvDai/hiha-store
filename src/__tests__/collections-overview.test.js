import React from 'react'

import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

import { CollectionOverview } from '../components/collections-overview/collections-overview.component'

describe('CollectionOverview', () => {
	it('should render without crashing', () => {
		const props = {
			collections: [
				{
					id: 10,
					imageUrl:
						'https://ik.imagekit.io/fireman35/images/0s3pdnc/adidas-nmd.png',
					name: 'Adidas NMD',
					price: 220,
				},
			],
		}
		const wrapper = shallow(<CollectionOverview {...props} />)

		expect(toJson(wrapper)).toMatchSnapshot()
	})
})
