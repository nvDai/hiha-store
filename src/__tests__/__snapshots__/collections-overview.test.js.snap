// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CollectionOverview should render without crashing 1`] = `
<div
  className="collection-overview"
>
  <withRouter(CollectionPreview)
    imageUrl="https://ik.imagekit.io/fireman35/images/0s3pdnc/adidas-nmd.png"
    key="10"
    name="Adidas NMD"
    price={220}
  />
</div>
`;
